### PR TITLE
feat(store): Add bol.com, mediamarkt.nl and fnac.be, updated amazon.nl and coolblue.com

### PR DESCRIPTION
**Pull request by johnneerdael**
feat: New stores added and items added to existing stores

### Description
Added 3 new stores: bol.com (NL), mediamarkt (NL) and fnac (BE)
Updated amazon (NL) and coolblue (NL) to add support for PS5

### CODE REVIEW

File: src/store/model/bol.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+export const Bol: Store = {
+	labels: {
+		captcha: {
+			container: 'body',
+			text: ['voer de karakters in die u hieronder ziet']
+		},
+		inStock: {
+			container: '.buy-block',
+			text: ['Op voorraad']
```
Comment (jef) [inline src/store/model/bol.ts:11]:
```suggestion
			container: 'div[data-test="buy-block__options"]',
			text: ['In winkelwagen']
```

Isn't this the correct verbiage?

File: src/store/model/bol.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		},
+		maxPrice: {
+			container: 'span[class*="PriceString"]',
```
Comment (jef) [inline src/store/model/bol.ts:14]:
```suggestion
			container: '.price-block__price',
```

or

```suggestion
			container: '.promo-price',
```

But not sure how reliable that is. `promo` makes it seem like it's temporary.

File: src/store/model/fnac.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	},
+	links: [
+		{
+			brand: 'test:brand',
+			cartUrl:
+				'https://secure.nl.fnac.be/basket/add?from=111&productId=14119956',
+			model: 'test:model',
+			series: 'test:series',
+			url:
+				'https://www.fnac.com/Console-Sony-PS5-Edition-Standard/a14119956/w-4'
```
Comment (jef) [inline src/store/model/fnac.ts:26]: Could you change to something that is a better test candidate? Something that is usually in stock?

File: src/store/model/mediamarkt-nl.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import {Store} from './store';
+
+export const MediamarktNL: Store = {
+	labels: {
+		inStock: {
+			container: '.offers',
+			text: ['Online op voorraad']
```
Comment (jef) [inline src/store/model/mediamarkt-nl.ts:7]:
```suggestion
			container: 'div.price-button',
			text: ['bestel nu']
```

Isn't this what you'd want?

image

**Review by jef — CHANGES_REQUESTED** [review]

**Comment by Revadike** [comment]
We need this added!

**Comment by Revadike** [comment]
@johnneerdael could you perhaps update this?

**Comment by jef** [comment]
@Revadike, if you want to take it over, I'll add!

**Pull request closed without merging**